public/index.html

```diff
@@ -7,6 +7,39 @@
     <title>2.3-photo-site</title>
 </head>
 <body>
-
+<nav>
+    <div class="left"></div>
+    <div class="right"></div>
+</nav>
+ <div class="header">
+     <img src="" alt="">
+     <p></p>
+ </div>
+ <ul class="posts">
```

Comment:
Don't use an `ul` and `li` here: figure is enough to set apart the picture (you may want to move your `.photo` class though.

`ul` elements should only have child `li` items and include similar if not the same layout of items.
Here you have footer in your `ul`.